public/controllers/agent/agents-preview.js

```diff
@@ -50,7 +50,7 @@ export class AgentsPreviewController {
     this.wazuhConfig = new WazuhConfig();
     this.errorInit = false;
     this.$window = $window;
-    this.timeService = TimeService;
+    this.formatUIDate = formatUIDate;
```

Comment:
This could be removed because you could use the `formatUIDate` directly without creating an instance variable of the component.

Reply:
Instance removed. Now, this function is called directly.